From: Carlos Maddela <e7appew@gmail.com>
Date: Fri, 23 Dec 2016 05:51:56 +1100
Subject: Don't allow doco to automatically fetch resources from the internet.

Description: Don't allow doco to automatically fetch resources from
 the internet.
 .
 Remove link to the newsfeed, to fix Lintian privacy-breach-generic
 warning. The news feed no longer works anyway.
Author: Carlos Maddela <e7appew@gmail.com>
Origin: vendor
Forwarded: not-needed
Last-Update: 2016-12-23
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 documentation/cs/index.html | 3 +--
 documentation/de/index.html | 3 +--
 documentation/en/index.html | 3 +--
 documentation/ru/index.html | 3 +--
 4 files changed, 4 insertions(+), 8 deletions(-)

diff --git a/documentation/cs/index.html b/documentation/cs/index.html
index 7fa1778..920d849 100644
--- a/documentation/cs/index.html
+++ b/documentation/cs/index.html
@@ -4,7 +4,6 @@
  <meta http-equiv="content-type" content="text/html; charset=UTF-8">
  <title>Přehled</title>
  <link rel="stylesheet" type="text/css" href="../include/dvdisaster.css">
-<link rel="alternate" type="application/atom+xml" href="http://dvdisaster.net/cs/feed/atom.xml" title="dvdisaster - novinky">
 </head>
 <body>
 
@@ -88,7 +87,7 @@
   <td style="background-color:#e0e0ff; width:20%;" valign="top">
     <table width="100%" cellpadding="10"><tr><td>
       <span class="fs"><b>Novinky</b></span>
-      <a href="http://dvdisaster.net/cs/feed/atom.xml"><img src="../images/atom16.png" alt="Subscribe to dvdisaster news feed" class="noborder"></a>
+      <img src="../images/atom16.png" alt="Subscribe to dvdisaster news feed" class="noborder">
       <table width="100%" cellpadding="0" cellspacing="0">
          <tr>
 	   <td class="hsep"></td>
diff --git a/documentation/de/index.html b/documentation/de/index.html
index 994dbcb..85f4076 100644
--- a/documentation/de/index.html
+++ b/documentation/de/index.html
@@ -4,7 +4,6 @@
  <meta http-equiv="content-type" content="text/html; charset=UTF-8">
  <title>Überblick</title>
  <link rel="stylesheet" type="text/css" href="../include/dvdisaster.css">
-<link rel="alternate" type="application/atom+xml" href="http://dvdisaster.net/de/feed/atom.xml" title="Neues von dvdisaster">
 </head>
 <body>
 
@@ -118,7 +117,7 @@ Solche Funktionen sind mit den Zielen und dem inneren Aufbau von dvdisaster nich
   <td style="background-color:#e0e0ff; width:20%;" valign="top">
     <table width="100%" cellpadding="10"><tr><td>
       <span class="fs"><b>Neuigkeiten</b></span>
-      <a href="http://dvdisaster.net/de/feed/atom.xml"><img src="../images/atom16.png" alt="Subscribe to dvdisaster news feed" class="noborder"></a>
+      <img src="../images/atom16.png" alt="Subscribe to dvdisaster news feed" class="noborder">
       <table width="100%" cellpadding="0" cellspacing="0">
          <tr>
 	   <td class="hsep"></td>
diff --git a/documentation/en/index.html b/documentation/en/index.html
index 32fe230..ff2a30f 100644
--- a/documentation/en/index.html
+++ b/documentation/en/index.html
@@ -4,7 +4,6 @@
  <meta http-equiv="content-type" content="text/html; charset=UTF-8">
  <title>Overview</title>
  <link rel="stylesheet" type="text/css" href="../include/dvdisaster.css">
-<link rel="alternate" type="application/atom+xml" href="http://dvdisaster.net/en/feed/atom.xml" title="dvdisaster news">
 </head>
 <body>
 
@@ -117,7 +116,7 @@ Such functions are outside the scope of dvdisaster's internal design and goals.<
   <td style="background-color:#e0e0ff; width:20%;" valign="top">
     <table width="100%" cellpadding="10"><tr><td>
       <span class="fs"><b>News</b></span>
-      <a href="http://dvdisaster.net/en/feed/atom.xml"><img src="../images/atom16.png" alt="Subscribe to dvdisaster news feed" class="noborder"></a>
+      <img src="../images/atom16.png" alt="Subscribe to dvdisaster news feed" class="noborder">
       <table width="100%" cellpadding="0" cellspacing="0">
          <tr>
 	   <td class="hsep"></td>
diff --git a/documentation/ru/index.html b/documentation/ru/index.html
index 1126100..d74caa1 100644
--- a/documentation/ru/index.html
+++ b/documentation/ru/index.html
@@ -4,7 +4,6 @@
  <meta http-equiv="content-type" content="text/html; charset=UTF-8">
  <title>Обзор</title>
  <link rel="stylesheet" type="text/css" href="../include/dvdisaster.css">
-<link rel="alternate" type="application/atom+xml" href="http://dvdisaster.net/ru/feed/atom.xml" title="dvdisaster news">
 </head>
 <body>
 
@@ -115,7 +114,7 @@ dvdisaster хранит данные на CD/DVD (<a href="qa10.html#media">по
   <td style="background-color:#e0e0ff; width:20%;" valign="top">
     <table width="100%" cellpadding="10"><tr><td>
       <span class="fs"><b>Новости</b></span>
-      <a href="http://dvdisaster.net/ru/feed/atom.xml"><img src="../images/atom16.png" alt="Subscribe to dvdisaster news feed" class="noborder"></a>
+      <img src="../images/atom16.png" alt="Subscribe to dvdisaster news feed" class="noborder">
       <table width="100%" cellpadding="0" cellspacing="0">
          <tr>
 	   <td class="hsep"></td>
